dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/ErrorDecoder.java

```diff
@@ -115,4 +115,10 @@ private static String toString(InputStream body) throws IOException {
             return CharStreams.toString(reader);
         }
     }
+
+    private static UnknownRemoteException toRemoteException(int code, QosException qos) {
```

Comment:
slight preference to inline this to reassure people there's nothing too crazy going on here